src/ml/neural_net/mps_layer_helper.h

```diff
@@ -11,20 +11,30 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+@class TCMPSConvolutionWeights;
+@class TCMPSInstanceNormDataLoader;
+
+@interface MPSCNNFullyConnectedNode(TCMPSLayerProperty)
```

Comment:
Just curious, why have a separate `TCMPSLayerProperty` category, separate from `TCMPSLayerHelper`?

Reply:
The helper category was to help create an `MPSCNNFullyConnectedNode` from the helper method. The `TCMPSLayerProperty` adds a property to the interface `MPSCNNFullyConnectedNode`. Functionally I though of them as two different things.

Reply:
Removed the separate category, refactored into one category.